client/extensions/woocommerce/app/order/order-details-table.js

```diff
@@ -0,0 +1,176 @@
+	}
+
+	getRefundedTotal = ( order ) => {
+		return order.refunds.reduce( ( total, i ) => total + ( i.total * 1 ), 0 );
+	}
+
+	recalculateRefund = () => {
+		if ( ! this.props.order ) {
+			return 0;
+		}
+		const subtotal = sum( this.state.quantities.map( ( q, i ) => {
+			if ( ! this.props.order.line_items[ i ] ) {
+				return 0;
+			}
+			return parseFloat( this.props.order.line_items[ i ].price ) * q;
+		} ) );
+		const total = subtotal + ( parseFloat( this.state.shippingTotal ) || 0 );
+		this.props.onChange( total );
+	}
+
+	onChange = ( event ) => {
+		if ( 'shipping_total' === event.target.name ) {
+			const shippingValue = event.target.value.replace( /[^0-9,.]/g, '' );
+			this.setState( { shippingTotal: shippingValue }, this.recalculateRefund );
+		} else {
+			const i = event.target.name.split( '-' )[ 1 ];
+			const newQuants = this.state.quantities;
+			newQuants[ i ] = event.target.value;
+			this.setState( { quantities: newQuants }, this.recalculateRefund );
+		}
+	}
+
+	renderTableHeader = () => {
+		const { translate } = this.props;
+		return (
+			<TableRow className="order__detail-header">
+				<TableItem isHeader className="order__detail-item-product">{ translate( 'Product' ) }</TableItem>
+				<TableItem isHeader className="order__detail-item-cost">{ translate( 'Cost' ) }</TableItem>
+				<TableItem isHeader className="order__detail-item-quantity">{ translate( 'Quantity' ) }</TableItem>
+				<TableItem isHeader className="order__detail-item-total">{ translate( 'Total' ) }</TableItem>
+			</TableRow>
+		);
+	}
+
+	renderOrderItems = ( item, i ) => {
+		const { isEditable, order, site } = this.props;
+		return (
+			<TableRow key={ i } className="order__detail-items">
+				<TableItem isRowHeader className="order__detail-item-product">
+					<a href={ getLink( `/store/product/:site/${ item.product_id }`, site ) } className="order__detail-item-link">
+						{ item.name }
+					</a>
+					<span className="order__detail-item-sku">{ item.sku }</span>
+				</TableItem>
+				<TableItem className="order__detail-item-cost">{ formatCurrency( item.price, order.currency ) || item.price }</TableItem>
+				<TableItem className="order__detail-item-quantity">
+					{ isEditable
+						? <FormTextInput
+							type="number"
+							name={ `quantity-${ i }` }
+							onChange={ this.onChange }
+							min="0"
+							max={ item.quantity }
+							value={ this.state.quantities[ i ] || 0 } />
+						: item.quantity
+					}
+				</TableItem>
+				<TableItem className="order__detail-item-total">{ formatCurrency( item.total, order.currency ) || item.total }</TableItem>
+			</TableRow>
+		);
+	}
+
+	renderRefundValue = () => {
+		const { order, translate } = this.props;
+		const refundValue = order.refunds.length ? this.getRefundedTotal( order ) : false;
+		if ( ! refundValue ) {
+			return null;
+		}
+
+		return (
+			<div className="order__details-total-refund">
+				<div className="order__details-totals-label">{ translate( 'Refunded' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-orders-refund)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).